cmd/launcher/options.go

```diff
@@ -83,6 +83,7 @@ func parseOptions(subcommandName string, args []string) (*launcher.Options, erro
 		flCompactDbMaxTx         = flagset.Int64("compactdb-max-tx", 65536, "Maximum transaction size used when compacting the internal DB")
 		flConfigFilePath         = flagset.String("config", defaultConfigFilePath, "config file to parse options from (optional)")
 		flExportTraces           = flagset.Bool("export_traces", false, "Whether to export traces")
+		flTraceSamplingRate      = flagset.Float64("trace_sampling_rate", 0.0, "What fraction of traces should be sampled")
```

Comment:
My sense is that doing this as an integer is going to be less code, and that there's very little gain from a float. OTOH, you wrote the code, so 🤷

Reply:
Unless I missed it, I don't think we support int flags yet in the `flags` package either, so it'd be the same amount of work: https://github.com/kolide/launcher/tree/main/pkg/agent/flags

Reply:
Ah, indeed.

Reply:
I just like ints more than floats. 😆